extensions/BugModal/template/en/default/bug_modal/field.html.tmpl

```diff
@@ -219,7 +219,7 @@ END;
 
         [% CASE constants.FIELD_TYPE_FREETEXT %]
           [%# normal input field %]
-          <input name="[% name FILTER html %]" id="[% name FILTER html %]" value="[% value FILTER html %]" [% aria_labelledby_html FILTER none %] [% "required" IF required %]>
+          <input name="[% name FILTER html %]" id="[% name FILTER html %]" value="[% value FILTER html %]" [% aria_labelledby_html FILTER none +%] [% " required" IF required %]>
         [% CASE constants.FIELD_TYPE_USER %]
```

Comment:
This is an unrelated change, which we can be fixed by doing a 'git fetch origin' followed by a `git rebase -i origin/master` and deleting the comment named `invalid html - missing space before "required" attribute`